pkgs/top-level/all-packages.nix

```diff
@@ -167,6 +167,10 @@ in
     deps = [ innoextract file-rename ]; }
     ../build-support/setup-hooks/gog-unpack.sh;
 
+  btnx-config = callPackage ../applications/misc/btnx-config {
+    inherit (gnome2) libglade;
```

Comment:
It is either just libglade or libglademm.

Reply:
In this case it is libglade. But for now I have set the PR in WIP because there has been an issue with the integration (application is looking for /etc/btnx content in a hardcoded way)